packages/compiler/core/type-utils.ts

```diff
@@ -75,3 +79,50 @@ export function isGlobalNamespace(
 ): namespace is Namespace & { name: ""; namespace: undefined } {
   return program.getGlobalNamespaceType() === namespace;
 }
+
+/**
+ * Check if the given type is contained in the specified namespace or, optionally, its child namespaces.
+ * @param type Type
+ * @param namespace Namespace
+ * @returns {boolean}
+ */
+export function isDefinedInNamespace(
```

Comment:
```suggestion
export function isDeclaredInNamespace(
```
does this make more sense?

Reply:
Sure, that's fine, I'll change it.